lib/jxl/enc_params.h

```diff
@@ -215,6 +215,7 @@ struct CompressParams {
   // modular mode options below
   ModularOptions options;
   int responsive = -1;
+  bool customsqueeze = false;
```

Comment:
Why don't we just put a SqueezeParams here too?
(I'd say a std::optional but c++11..)

Reply:
A std::vector<SqueezeParams>? I guess that is the most flexible way to do it, yes. We likely will not really want to expose it like that in the encoder api though :)

Reply:
right, a vector of params would work too - empty means default.